docs/specification.md

```diff
@@ -0,0 +1,155 @@
+# Homebase Lite Spec Draft
+
+This spec draft is for the proposed Tezos smart contract to be used as the on-chain component of a new lightweight Homebase tool.
```

Comment:
No need to make this a "draft" anymore, this was approved and if there are any more doubts we should clarify them now or whenever they appear :slightly_smiling_face: 

For clarity I would replace this with a line that says that the contract is under active development and there could be changes because of that.